custom_components/tahoma/tahoma_device.py

```diff
@@ -109,10 +91,27 @@ def device_info(self) -> Dict[str, Any]:
         manufacturer = self.select_state(CORE_MANUFACTURER_NAME_STATE) or "Somfy"
         model = self.select_state(CORE_MODEL_STATE) or self.device.widget
 
+        name = self.name
+        device_url = self.device_url
+        # Some devices, such as the Smart Thermostat have several devices in one physical device,
+        # with same device URL, terminated by '#' and a number.
+        # We use the base url as a device unique id.
+        if "#" in self.device_url:
```

Comment:
I can't test this code, but what if the main device is not mapped in HA? (for example when the main Climate device is not implemented yet). How will this work?

Reply:
Added in separate PR.